pkg/services/alerting/rule.go

```diff
@@ -137,7 +138,8 @@ func NewRuleFromDBAlert(ruleDef *models.Alert) (*Rule, error) {
 	for _, v := range ruleDef.Settings.Get("notifications").MustArray() {
 		jsonModel := simplejson.NewFromAny(v)
 		if id, err := jsonModel.Get("id").Int64(); err == nil {
-			model.Notifications = append(model.Notifications, fmt.Sprintf("%09d", id))
+			maybeUID := translateNotificationIDToUID(id, ruleDef.OrgId)
+			model.Notifications = append(model.Notifications, maybeUID...)
```

Comment:
If `translateNotificationIDToUID `returns `string` instead of `[]string`, following my suggestion, then you won't need `...`.

Reply:
done